src/phoenix/db/engines.py

```diff
@@ -118,7 +120,14 @@ def async_creator() -> aiosqlite.Connection:
         else:
             asyncio.create_task(init_models(engine))
     else:
-        migrate_in_thread(engine.url)
+        migrate_in_thread(
+            sqlalchemy.create_engine(
+                url=url.set(drivername="sqlite"),
+                echo=Settings.log_migrations,
+                json_serializer=_dumps,
+                creator=lambda: sqlean.connect(f"file:{database}", uri=True),
+            )
+        )
```

Comment:
I'm confused as to how this works - isn't the driver inferred from the URL? How can this be hard coded to sqlite?

Reply:
Sync'd offline - postgres is a different codepath below